javascript/javascript1/week3/exercises.js

```diff
@@ -0,0 +1,267 @@
+    }% of my life`
+  );
+  return watchingGameOfThronesPercentageOfLife;
+}
+function calculateWatchingOrvillePercentageOfLife() {
+  const myDurationOfWatchingOrvilleInMinutes = (0 * 24 + 18) * 60 + 54;
+  console.log(myDurationOfWatchingOrvilleInMinutes);
+  watchingOrvillePercentageOfLife =
+    (myDurationOfWatchingOrvilleInMinutes * 100) / averageLifespanInMinutes;
+  console.log(
+    `The Orville took ${
+      Math.round(watchingOrvillePercentageOfLife * 10000) / 10000
+    }% of my life`
+  );
+  return watchingOrvillePercentageOfLife;
+}
+
+function calculateWatchingSquidGamePercentageOfLife() {
+  const myDurationOfWatchingSquidGameInMinutes = (0 * 24 + 8) * 60 + 6;
+  console.log(myDurationOfWatchingSquidGameInMinutes);
+  watchingSquidGamePercentageOfLife =
+    (myDurationOfWatchingSquidGameInMinutes * 100) / averageLifespanInMinutes;
+  console.log(
+    `Squid game took ${
+      Math.round(watchingSquidGamePercentageOfLife * 10000) / 10000
+    }% of my life`
+  );
+  return watchingSquidGamePercentageOfLife;
+}
+
+function calculateWatchingSeriesPercentageOfLife() {
+  const myDurationOfWatchingTvSeriesInPercentageOfLife =
+    watchingGameOfThronesPercentageOfLife +
+    watchingOrvillePercentageOfLife +
+    watchingSquidGamePercentageOfLife;
+  console.log(
+    `In total that is ${
+      Math.round(myDurationOfWatchingTvSeriesInPercentageOfLife * 10000) / 10000
+    }% of my life`
+  );
+  return myDurationOfWatchingTvSeriesInPercentageOfLife;
+}
+
+sumSeriesDurations();
+calculateMinutes();
+calculateHoursAndDays();
+logSumWatching();
+calculateWatchingGameOfThronesPercentageOfLife();
+calculateWatchingOrvillePercentageOfLife();
+calculateWatchingSquidGamePercentageOfLife();
+calculateWatchingSeriesPercentageOfLife();
+
+// Step 3: Smart-ease - Back to the basics!
+
+const notes = [];
+
+function saveNote(content, id) {
+  // write some code here
+  notes.push({ content, id });
+  return notes;
+}
+
+function getNote(id) {
+  if (id > -1 && id <= notes.length + 1) {
+    for (let i = 1; i < notes.length + 1; i++) {
+      if (id === i) {
+        console.log(notes[i - 1]);
+        return notes[i - 1];
+      }
+    }
+    const idError =
+      "The note with this id doesn't exist! Please enter a correct id";
+    console.log(idError);
+    return idError;
+  } else if (typeof id === "string") {
+    const formatError =
+      "This id format isn't correct. Please enter a correct id";
+    console.log(formatError);
+    return formatError;
+  }
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("Make homework", 3);
+
+const firstNote = getNote(1);
+const secondNote = getNote(3);
+const thirdNote = getNote(2);
+const fourthNote = getNote(null);
+const fifthNote = getNote("qwerty");
+const sixthNote = getNote(4);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+function logOutNotesFormatted(id) {
```

Comment:
this was supposed to log out all notes, i think, but the function is correct and the change is easy to make :)